packages/core/src/transport/sendWithRetryStrategy.ts

```diff
@@ -0,0 +1,174 @@
+import { addTelemetryDebug } from '../domain/telemetry'
+import { monitor } from '../tools/monitor'
+import { ONE_KIBI_BYTE, ONE_MEBI_BYTE, ONE_SECOND } from '../tools/utils'
+import type { Payload, HttpResponse } from './httpRequest'
+
+export const MAX_ONGOING_BYTES_COUNT = 80 * ONE_KIBI_BYTE
+export const MAX_ONGOING_REQUESTS = 32
+export const MAX_QUEUE_BYTES_COUNT = 3 * ONE_MEBI_BYTE
+export const MAX_BACKOFF_TIME = 256 * ONE_SECOND
+export const INITIAL_BACKOFF_TIME = ONE_SECOND
+
+enum TransportStatus {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we meant to enforce using `const enum` to help minification when possible, but the ESLint configuration is incorrect and fails to detect this. I fixed it in https://github.com/DataDog/browser-sdk/pull/1709

```suggestion
const enum TransportStatus {
```